python/lsst/ts/genericcamera/driver/__init__.py

```diff
@@ -20,6 +20,11 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 from .basecamera import *
+
+try:
```

Comment:
Can you add a warning in the case it cannot import the camera?

Reply:
I thought about that, but the warning will show up all the time if the library is not installed. If you're OK with that, I'll add the warning.

Reply:
Warning added.